test.py

```diff
@@ -53,14 +53,6 @@ def test_manager(self):
         self.assertEqual(r, "jaja-lol-a")
 
         txt = 'jaja---lol-méméméoo--a'
-        r = slugify(txt, max_length=19, word_boundary=True)
```

Comment:
Hmm, I thought about it. The situation was described in merge request comment (len=17 and len=19). Should I add a new parameter to function or it is a bug?